ui/pages/confirmations/components/confirm/info/info.tsx

```diff
@@ -23,6 +23,7 @@ const Info = () => {
         return TypedSignInfo;
       },
       [TransactionType.contractInteraction]: () => ContractInteractionInfo,
```

Comment:
Outside of the scope of this PR, but does it make conceptual sense for use to key the above signature components using `TransactionType` rather than `ApprovalType`?

We could theoretically default all confirmations with the `Transaction` approval type to our `ContractInteractionInfo`, and have a separate map for any `TransactionType` specific overrides?